website/content/vagrant-cloud/support.mdx

```diff
@@ -33,7 +33,7 @@ it provides, there may be information lacking in the documentation.
 Vagrant Cloud documentation is open source.
 If you'd like to improve or correct the documentation,
 please submit a pull-request to the
-[Vagrant project on GitHub](https://github.com/hashicorp/vagrant/tree/main/website/pages/docs/vagrant-cloud/).
+[Vagrant project on GitHub](https://github.com/hashicorp/vagrant/tree/main/website/pages/vagrant-cloud/).
```

Comment:
Think the directory structure has changed a bit -  both URLs lead to a 404. This new path seems to work
```suggestion
[Vagrant project on GitHub](https://github.com/hashicorp/vagrant/tree/main/website/content/vagrant-cloud/).
```

Reply:
Good catch!